### ISSUE TITLE
Running model.fit() multiple times on batches for DeepFM

### ISSUE THREAD

**Issue by leoanghelescu** [shenweichen/DeepCTR#45]
**Describe the question(问题描述)**

I am using the DeepFM module and have a lot of data. I would like to run the model in batches, using the model.train function()

        history = model.fit(train_model_input
                            , train[target].values
                            , batch_size=1024
                            , epochs=1
                            , verbose=2
                            , validation_split=0.05)

This seems to work iteratively, my loss does go down, my AUC does go up, but can we confirm this please ? I have read conflicting articles about this.

Also, I am wondering about the consistency of the embedding across batches. Given that embedding representation depends on the composition of a dataset, does it makes sense/does it work, if I incrementally train the model using calls on model.train() on a sequence of files?

Please let me know. Absolutely love this library so far I must say too 👍 

**Operating environment(运行环境):**
 - python version [3.5]
 - tensorflow version [1.4.0]
 - deepctr version [latest]

**Comment by shenweichen** [comment]
yes,it makes sense when training model with `model.train()` iteratively.

**Comment by leoanghelescu** [comment]
Thanks!
